### PR TITLE
basic zipcode component #2

### PR DESCRIPTION
**Pull request by ivinjose**

### CODE REVIEW

File: app/scripts/ui/zipcode.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+var Button  = require('react-bootstrap/Button');
+var Input   = require('react-bootstrap/Input');
+
+var Zipcode = React.createClass({
+    displayName: 'Zipcode',
+    render: function(){
+        return(
+            <Modal title="Enter your zipcode">
+                <div className="modal-body" id="zipcode-body">
+                    <Input
```
Comment (shidhincr) [inline app/scripts/ui/zipcode.js]: This we might need to change to ZipcodeInput component later, as the same is used in search page

**Comment by shidhincr** [comment]
@ivinjose Is this just the layout ? Should we need another ticket for the functionality or shall we add it here ?
 
What do you think ?

File: app/scripts/ui/zipcode.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+var React   = require('react');
+var Modal   = require('react-bootstrap/Modal');
+var Button  = require('react-bootstrap/Button');
+var Input   = require('react-bootstrap/Input');
+
+var Zipcode = React.createClass({
+    displayName: 'Zipcode',
+    render: function(){
+        return(
+            <Modal title="Enter your zipcode">
```
Comment (shidhincr) [inline app/scripts/ui/zipcode.js]: Let's not use '-' in the id string. We'll set up an editor config soon

Comment (shidhincr) [inline app/scripts/ui/zipcode.js]: Can you make it like the existing one in lookaround.herokuapp.com ?

Comment (ivinjose) [inline app/scripts/ui/zipcode.js]: Not having '-' in the id string, is it because of some naming convention we are going to follow?

Comment (shidhincr) [inline app/scripts/ui/zipcode.js]: Yeah.. naming conventions are better

**Comment by shidhincr** [comment]
Let me check this. If the functionality is a separate ticket, we can merge this

**Comment by shidhincr** [comment]
@ivinjose I did couple of UI fixes. Merging this branch now. Let's work on the other tickets

**Comment by shidhincr** [comment]
closed issue #16

**Pull request merged**